.changeset/dull-mugs-fail.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/plugin-catalog-backend-module-gerrit': minor
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-gerrit': patch
```

This package is 0.x and this isn't a breaking change right?

Reply:
I marked this as a breaking change since it changed to what the `branch` configuration parameter defaults to:

> **BREAKING** The optional `branch` configuration parameter now defaults to the default branch of the project (where `HEAD` points to).
This parameter was previously using `master` as the default value. In most cases this change should be transparent as Gerrit defaults to using `master`.

But let me know if you want me to use a patch instead (and perhaps remove the breaking tag then)?